### PR TITLE
Add in CDN_* defines 

### PR DESCRIPTION
**Pull request by chenss3**
Fixes: #1301

### CODE REVIEW

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -34862,6 +34862,52 @@
       }
     ]
   },
+  {
+    "name": "CDN_CONSTANTS",
```
Comment (riverar) [inline generation/WinSDK/enums.json:34866]: Perhaps better named this? (https://learn.microsoft.com/windows/win32/dlgbox/common-dialog-box-notifications)

```suggestion
    "name": "COMMON_DIALOG_NOTIFICATIONS",
```

Comment (mikebattista) [reply generation/WinSDK/enums.json:34866]: Should be singular not plural if this isn't a flags enum.

File: generation/WinSDK/requiredNamespacesForNames.rsp [hunk 2]
```diff
@@ -2452,4 +2452,5 @@ UIA_HEADINGLEVEL_ID=Windows.Win32.UI.Accessibility
 UIA_LANDMARKTYPE_ID=Windows.Win32.UI.Accessibility
 UIA_METADATA_ID=Windows.Win32.UI.Accessibility
 UIA_STYLE_ID=Windows.Win32.UI.Accessibility
-UIA_TEXTATTRIBUTE_ID=Windows.Win32.UI.Accessibility
\ No newline at end of file
+UIA_TEXTATTRIBUTE_ID=Windows.Win32.UI.Accessibility
+CDN_CONSTANTS=Windows.Win32.UI.Controls.Dialogs
```
Comment (mikebattista) [inline generation/WinSDK/requiredNamespacesForNames.rsp:2456]: Move the namespace to enums.json.

**Review by mikebattista — CHANGES_REQUESTED** [review]
See comments.

**Comment by elachlan** [comment]
Merge conflict.

**Review by mikebattista — APPROVED** [review]

**Pull request merged**